src/nghttp.cc

```diff
@@ -687,7 +687,9 @@ void HttpClient::disconnect() {
   state = ClientState::IDLE;
 
   for (auto req = std::begin(reqvec); req != std::end(reqvec); ++req) {
-    (*req)->continue_timer->stop();
+    if ((*req)->continue_timer) {
```

Comment:
On this note, is there a test suite for the helper utilities? I missed this typo because I ran my usual set of tests against servers, but completely forgot to run without the `--expect-continue` flag...